File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2424.go
package main
 
import (
    "fmt"
)

func main() {
    var x, y int;

    fmt.Scanf("%d %d", &x, &y);

    if(x < 0 || x > 432 || y < 0 || y > 468){ fmt.Printf("fora\n");
    }else{ fmt.Printf("dentro\n"); }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2424.c
#include <stdio.h>

int main(){
    int x, y;
    scanf("%d %d", &x, &y);
    if(x < 0 || x > 432 || y < 0 || y > 468) printf("fora\n");
    else printf("dentro\n");
    
    return 0;
}